# Message for multiple tracked humans
std_msgs/Header header

# Array of tracked humans
TrackedHuman[] humans

# Frame reference (map, odom, etc)
string frame_id